userspace/libsinsp/event.cpp

```diff
@@ -1019,9 +1019,14 @@ const char* sinsp_evt::get_param_as_str(uint32_t id, const char** resolved_str,
 	case PT_FSRELPATH:
 	{
 		std::string_view path = param->as<std::string_view>();
+		if(path.length() + 1 > m_paramstr_storage.size())
+		{
+			m_paramstr_storage.resize(path.length() + 1);
+		}
+
 		strcpy_sanitized(&m_paramstr_storage[0],
 			path.data(),
-			std::min(path.size() + 1, m_paramstr_storage.size()));
+			path.length() + 1);
```

Comment:
now path.length()+1 should be always lower or equal than `m_paramstr_storage.size()`